plugins/content-manager/src/main/java/com/wazuh/contentmanager/cti/catalog/index/ContentIndex.java

```diff
@@ -84,6 +85,8 @@ public class ContentIndex {
         "parse|event.original", "parse|message", "normalize"
     );
 
+    private enum spaceName { free, paid, custom }
```

Comment:
Enum are usually uppercase.

Move the space enum to a dedicated file / class. File should be located under the `catalog.model` package.

```java
enum Space {
    FREE, PAID, CUSTOM
}